tests/libres_tests/res/util/test_subprocess.py

```diff
@@ -45,6 +46,7 @@ def test_await_process_tee(self):
             fh.write(bytearray(os.urandom(_maxBytes)))
 
         with open("a", "wb") as a_fh, open("b", "wb") as b_fh:
+            # pylint: disable=consider-using-with
```

Comment:
Does the comment ` # await_process_tee() ensures the process is terminated.` fit here?